recipes/nfft/meta.yaml

```diff
@@ -0,0 +1,49 @@
+#{% set version = "3.3.1" %}
+{% set version = "3.2.4" %}
+
+package:
+  name: nfft
+  version: {{ version }}
+
+source:
+  fn: nfft-{{ version }}.tar.gz
+  # url: https://github.com/NFFT/nfft/releases/download/{{ version }}/nfft-{{ version }}.tar.gz
+  # md5: 8c0516e3252ff39a245e62f1a59d0165
+  url: https://www-user.tu-chemnitz.de/~potts/nfft/download/nfft-{{ version }}.tar.gz
+  md5: 8d9b1b708236297397ed1b70cace8b7b
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - fftw
+    - gcc  # [osx]
```

Comment:
It does. The problem is that it you would ship a broken package without `libgmop` and or you would need to request the end user to install that from her/his package manager and expect it to be compatible. So the current choice, to be on the safe side, is to use the default channel one and add `libgcc` are a run time dependency.